pkg/kubectl/cmd/util/printing.go

```diff
@@ -30,6 +33,39 @@ func AddPrinterFlags(cmd *cobra.Command) {
 	cmd.Flags().StringP("template", "t", "", "Template string or path to template file to use when -o=template or -o=templatefile.  The template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview]")
 }
 
+// AddOutputFlagsForMutation adds output related flags to a command. Used by mutations only.
+func AddOutputFlagsForMutation(cmd *cobra.Command) {
+	cmd.Flags().StringP("output", "o", "", "Output mode. Use \"-o name\" for showing only kind/name.")
+}
+
+// PrintSuccess prints message after finishing mutating operations
+func PrintSuccess(simple bool, out io.Writer, resource string, name string, operation string) {
+	if simple {
+		// -o name: prints kind/name
```

Comment:
I don't mind telling users it's kind, but internally the distinction matters.  "prints resource/name"